examples/workdir/inventory/hosts.yml

```diff
@@ -1,4 +1,8 @@
 ---
+localhost:
+  vars:
+   ansible_python_interpreter: /usr/bin/python3
```

Comment:
cant we set this in the inventory directly instead?

Reply:
@Itxaka there is no inventory file for "localhost" as far as I can see.and I couldn't get this to work in all cases. is there a problem with listing this in the tasks? the problem is that if you set it in the inventory it didn't have an "effect" in all cases, leaving a weird mixture of python2 and python3 being executed, which fails if python2 is not there.

Reply:
not a problem, but the inventory seems like the logical place for this IMO.

It not working in all cases kind of worries me :(

Reply:
That worries me too. I thought we found all the places we confused targeting localhost vs targeting the deployer while getting the job green in jenkins, but maybe there are still some mistakes?

Reply:
@jesusaurus yeah, but thats a matter of "our CI sucks" and we have an obvious solution here: extending CI coverage. If we agree the change is a good one and ci passes then we should merge it and deal with the fallout, including upgrading the ci coverage.

Reply:
This, behind the scenes, will create an explicit localhost entry in the ansible inventory, instead of relying on the host named "localhost" implicitly created by ansible.

For that implicit inventory host, ansible is using its own python version interpreter, which should be okay, assuming you install ansible under python3. This whole thing shouldn't therefore be used.

Reply:
@evrardjp I have removed the explicit localhost entry and moved the variable setting to other places.